{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -3,29 +3,39 @@
 language: python
 
 python:
-  - "3.5"
+  - "3.6"
 
 env: {% if '1.8' in cookiecutter.django_versions %}
+  - TOX_ENV=py36-django-18
   - TOX_ENV=py35-django-18
   - TOX_ENV=py34-django-18
   - TOX_ENV=py33-django-18
```

Comment:
Given that [3.3 has reached EOL](https://devguide.python.org/#status-of-python-branches), I suppose it could be dropped?

Reply:
Yes, we should drop 3.3. In fact, I argue we should drop everything but 3.6

Reply:
tho 3.5 still is the most up to date version u can install (with the default package manager) in some linux distros .. i also think, that pushing everyone to install 3.6 is worth it! 
f-expressions-❤

Reply:
By the same argument would people suggest should we also drop tox/travis checks on django 1.9, 1.10?   (and 1.8 once it goes out of LTS support in April?)

Reply:
I've dropped py33.

I guess we can drop Django<1.11 in a week or so after 1.8 goes out of LTS.